pkgs/servers/web-apps/phppgadmin/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub } :
+
+stdenv.mkDerivation rec {
+  name = "phppgadmin-${version}";
+  version = "7.13.0";
+
+  src = fetchFromGitHub {
+    owner  = "phppgadmin";
+    repo   = "phppgadmin";
+    rev    = "REL_7-13-0";
```

Comment:
```suggestion
    rev    = "REL_${replaceStrings [ "." ] [ "-" ] version}";
```

Reply:
i have changed it to your suggestion and force pushed it to my branch.
didn't know of the "commit suggestion" function. i hope that's ok?

Reply:
Both ways work work for me and I don't have strong feelings when those little changes are not attached to my name.